create-mern-ts-app.test.js

```diff
@@ -0,0 +1,3 @@
+test("test for truthy", () => {
+    expect(true).toBeTruthy();
+})
```

Comment:
Not sure if eslint complained here for you, but the rules specify:
* use of single quotes (line 1 uses double quotes)
* 2-space indentation (line 2 has 4 spaces)
* required semicolon (line 3 is missing semicolon)

Sorry to annoy you with that, I'll setup some prettier script in the future to avoid that. But could you please fix before I merge?

Reply:
Another thing, the `jest` environment needs to be configured with `eslint`.

You have to simply add this lines in `.eslintrc`
```json
"env": {
    "jest": true
}
```

Reply:
@Fabianopb No problem and thanks for the feedback. My VSCOde eslint did not complain. I will address those issues and update this PR.

Reply:
@Fabianopb Can you review this PR and let me know if there are any other issues that needs to be addressed ?

Thanks,
Jayadev

Reply:
That's really great @Jayadev6191 , thanks a lot for your contribution! I'll merge it now!